chemprop/uncertainty/predictor.py

```diff
@@ -91,12 +128,45 @@ def _calc_prediction_uncertainty(
 
 @UncertaintyPredictorRegistry.register("dropout")
 class DropoutPredictor(UncertaintyPredictor):
-    def _calc_prediction_uncertainty(
-        self, dataloader: DataLoader, models: Iterable[MPNN], trainer: pl.Trainer
-    ) -> Tensor:
-        ...
-        return
-
+    """
+    Class that creates an artificial ensemble of models by applying monte carlo dropout to the loaded
+    model parameters. Predicts uncertainty for predictions based on the variance in predictions among
+    an ensemble's submodels.
+    """
+
+    def __init__(self, sampling_size, dropout_p):
+        self.sampling_size = sampling_size
+        self.dropout_p = dropout_p
+
+    def _calc_prediction_uncertainty(self, dataloader, models, trainer) -> Tensor:
+        # TODO: uses first model if multiple are given, should throw error if multiple are given.
```

Comment:
I am fine with either providing a warning message and using only the first model or raising an error.